keentools_facebuilder/preferences/operators.py

```diff
@@ -86,10 +116,11 @@ def draw(self, context):
         content = ["You can download",
                    "Core library from ",
                    "our site: keentools.io/downloads"]
+        import time
         col = layout.column()
         col.scale_y = Config.text_scale_y
         for c in content:
-            col.label(text=c)
+            col.label(text=str(self.filepath))
```

Comment:
It looks like an error here, why output the same thing in a loop instead of message